src/Operators/spacings_and_areas_and_volumes.jl

```diff
@@ -140,10 +140,13 @@ using Oceananigans.Grids: Flat
 
 @inline Δxᶜᶠᵃ(i, j, k, grid::RegularLatitudeLongitudeGrid) = @inbounds grid.radius * hack_cosd(grid.φᵃᶠᵃ[j]) * deg2rad(grid.Δλ)
 @inline Δxᶠᶜᵃ(i, j, k, grid::RegularLatitudeLongitudeGrid) = @inbounds grid.radius * hack_cosd(grid.φᵃᶜᵃ[j]) * deg2rad(grid.Δλ)
+@inline Δxᶠᵃᵃ(i, j, k, grid::RegularLatitudeLongitudeGrid) = Δxᶠᶜᵃ(i, j, k, grid)
```

Comment:
I don't think we want to add this operator --- this could lead to bugs? I think we want to only use operators that fully specify the horizontal location for curvilinear grids.